packages/dev/docs/pages/react-spectrum/styling.mdx

```diff
@@ -174,26 +174,41 @@ All of the properties supported by `View` are listed below. All style properties
 
 Style props that accept colors, such as backgrounds and borders, only accept Spectrum defined color values.
 In addition to ensuring consistency across products, Spectrum colors automatically adapt to color
-scheme changes, e.g. dark mode.
+scheme changes, e.g. dark mode. Learn more about color on the [Spectrum website](https://spectrum.adobe.com/page/color-fundamentals/).
+
+There are two versions of Spectrum colors available: v5, and v6. v6 offers an expanded color palette, and we
+recommend using it for new projects. However, v5 is still the default for backward compatibility. You can
+opt into v6 by setting the `colorVersion` prop on the [View](View.html) component to `6`. A [migration guide](https://spectrum.adobe.com/static/misc/Colors_v6.0_Migration-Guide.xd)
```

Comment:
So for my own knowledge, this is saying that users need to wrap their app/components in a View that has `colorVersion={6}` in order to get the new colors? Or am I misunderstanding this and they would just need to have a View with `colorVersion={6}`? Also what do you think about linking back to this page from View's `colorVersion` prop description?

Reply:
No, they have to put `colorVersion={6}` on the `View` component for the `backgroundColor`/`borderColor` etc. props to be v6. It doesn't affect children.

Reply:
Ahhh, ok makes sense

Reply:
Devon's answer to Daniel doesn't match how I read that paragraph. His question and my confusion suggestion this should be rewritten to avoid confusion. Maybe my above suggestion is enough.